cmd/root.go

```diff
@@ -185,7 +196,18 @@ func parseConfig(cmd *cobra.Command, conf *proxy.Config, args []string) error {
 				return newBadCommandError(fmt.Sprintf("could not parse query: %q", res[1]))
 			}
 
-			if a, ok := q["address"]; ok {
+			a, aok := q["address"]
```

Comment:
Why did you move these up? They seem cleaner scoped to the if statement.

Reply:
Just below, I check that both address and unix, or address and port have not been set.